client/my-sites/all-sites-icon/index.jsx

```diff
@@ -31,13 +31,13 @@ export default React.createClass( {
 	getIcons() {
 		let sites = union( this.getSitesWithIcons(), this.getMaxSites() ).slice( 0, MAX_ICONS );
 		return sites.map( function( site ) {
-			return <SiteIcon site={ site } key={ site.ID + '-icon' } size={ 25 } />;
+			return <SiteIcon site={ site } key={ site.ID + '-icon' } size={ 16 } />;
 		} );
 	},
 
 	render() {
 		const icons = this.getIcons();
-		const classes = 'all-sites-icon is-shape-' + this.getMaxSites().length;
+		const classes = `all-sites-icon has-${this.getMaxSites().length}-icons`;
```

Comment:
Yes, if this goes through I'll remove all the old CSS, but I'll keep the `has-` class because it seems valuable.